neuralmonkey/decoders/ctc_decoder.py

```diff
@@ -93,8 +101,7 @@ def logits(self) -> tf.Tensor:
         weights = get_variable(
             name="state_to_word_W",
             shape=[encoder_states.shape[2], vocabulary_size + 1],
-            initializer=tf.variance_scaling_initializer(
-                mode="fan_avg", distribution="uniform"))
+            initializer=tf.random_uniform_initializer(-0.5, 0.5))
```

Comment:
Proc ta zmena tady?

Reply:
Nevím. IHMO protože cheme, aby to bylo stejně jako v autoregresivním dekodéru před půl rokem. Tohle zrovna psal @jindrahelcl

Reply:
já nevim to chce voměřit. to nový je vskutku ten důvod, co psal @jlibovicky